### PR TITLE
support set summary on TFEstimator, KerasModel

### PR DESCRIPTION
**Pull request by Bob-Chou**
Add `set_train_summary` and `set_val_summary` to `TFEstimator` and `KerasModel`.

### CODE REVIEW

**Comment by yangw1234** [comment]
please also add the documents

**Comment by hkvision** [comment]
LGTM

**Comment by Bob-Chou** [comment]
Related doc has been updated

File: docs/docs/ProgrammingGuide/visualization.md [hunk 1]
```diff
@@ -68,6 +68,39 @@ If logdir is absolute path started with `/`, like `/user/logdir/inpcetion_log`,
 
 If logdir is URI started with `hdfs://[host:port]/`, like `hdfs://172.168.0.101:8020/user/logdir/inpcetion_log`, the log will be stored to HDFS;  
 
+### **Generating summary in KerasModel**
+****python****
```
Comment (hkvision) [inline docs/docs/ProgrammingGuide/visualization.md]: Please update these according to https://github.com/intel-analytics/analytics-zoo/pull/1520/files

File: pyzoo/zoo/tfpark/estimator.py [hunk 2]
```diff
@@ -185,6 +187,12 @@ def set_gradient_clipping_by_l2_norm(self, clip_norm):
         """
         self.gradient_clipping_norm = clip_norm
 
+    def set_train_summary(self, summary):
+        self.train_summary = summary
```
Comment (hkvision) [inline pyzoo/zoo/tfpark/estimator.py]: Also add some inline doc in these methods.

**Comment by hkvision** [comment]
jenkins passed: http://10.239.47.211:18888/job/ZOO-PR-Validation/2450/

**Pull request merged**